docs/release.md

```diff
@@ -0,0 +1,13 @@
+## Release
+
+The steps:
+
+1.  [draft](https://github.com/zigbee-alliance/distributed-compliance-ledger/releases/new) and publish a new GitHub release:
+    *   specify a new release tag based on a new planned version of DCLedger in the format `v<version>` (e.g. `v1.2.3` for the version `1.2.3`)
+    *   verify the branch/commit target for the release: usually it should be `master` but other targets are possible as well
+    *   put release notes
+2.  once published [release pipeline](https://github.com/zigbee-alliance/distributed-compliance-ledger/actions/workflows/release.yml) is triggered:
+    *   it builds `dcld` binary on `ubuntu-20.04` and attaches the asset to the GitHub release
```

Comment:
I think it would be great to build for Mac as well, as this is one of popular platforms for CLI.